homework_7/statues.py

```diff
@@ -0,0 +1,16 @@
+"""Statues"""
+
+import random
+STATUES = []
+for i in range(4):
+    STATUE = 0
+    while STATUE == 0 or STATUE in STATUES:
+        STATUE = random.randint(1, 10)
+    STATUES.append(STATUE)
+STATUES.sort()
```

Comment:
you do not need to sort, the easiest way is to get min() and max() values and iterate from min to max